doc/ircd.8

```diff
@@ -1,5 +1,4 @@
 .\" @(#)ircd.8 2.0 22 April 2004
-.\" $Id: ircd.8 6 2005-09-10 01:02:21Z nenolod $
 .TH IRCD 8 "ircd-ratbox 22 April 2004
 .SH NAME
 ircd \- The Internet Relay Chat Program Server 
```

Comment:
ARPANET-compatible relay chat software program server for VAX-compatible timesharing mainframes